packages/region-config-resolver/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./extensions";
+
+export * from "./regionConfig";
```

Comment:
nit: remove extra new line.
```suggestion
export * from "./extensions";
export * from "./regionConfig";
```